packages/pdbresym.vm/pdbresym.vm.nuspec

```diff
@@ -2,13 +2,14 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>pdbresym.vm</id>
-    <version>1.3.6.20250219</version>
+    <version>1.3.6.20250509</version>
     <authors>Stephen Eckels</authors>
-    <description>Download PDBs</description>
+    <description>PDBReSym simplifies and optimizes interacting with the Microsoft Symbol Server to download PDBs. Requires substantial disk space.</description>
```

Comment:
It is pdbs.pdbresym.vm (which downloads the symbols using PDBReSym) and not pdbresym.vm that requires substantial disk space. Can you please improve the description of both pdbresym.vm and pdbs.pdbresym.vm to this PR? I think pdbs.pdbresym.vm does not need a project URL.